tests/integration/test_clip_text.py

```diff
@@ -78,45 +84,25 @@ def validate_request_traversal(resp):
         f.post(on='/test', inputs=docs, parameters={'traversal_path': ['cc']}, on_done=validate_request_traversal)
 
 
-def test_custom_processing():
-    f = Flow().add(uses=ClipTextEncoder)
-    with f:
-        result1 = f.post(on='/test', inputs=[Document(text='random txt')],
-                         return_results=True)
-
-    f = Flow().add(uses={
-        'jtype': ClipTextEncoder.__name__,
-        'with': {
-            'use_default_preprocessing': False,
-        }
-    })
-
-    with f:
-        result2 = f.post(on='/test', inputs=[Document(text=np.ones((224, 224, 3), dtype=np.float32))],
-                         return_results=True)
-
-    assert result1[0].docs[0].embedding is not None
-    assert result2[0].docs[0].embedding is not None
-    np.testing.assert_array_compare(operator.__ne__, result1[0].docs[0].embedding, result2[0].docs[0].embedding)
-
-
 def test_no_documents():
+    def validate_response(resp):
+        assert resp.status.code == 0  # SUCCESS
+
     with Flow().add(uses=ClipTextEncoder) as f:
-        results = f.post(on='/test', inputs=[], return_results=True)
-        assert results[0].status.code == 0  # SUCCESS
+        f.post(on='/test', inputs=[], on_done=validate_response)
 
 
 def test_clip_data():
     docs = []
-    fruits = ['apple', 'banana']
-    for fruit in fruits:
-        docs.append(Document(text=fruit))
+    entities = ['apple', 'banana1', 'banana2', 'studio', 'satelite', 'airplane']
```

Comment:
this test makes no sense. Clip should encode sentences not single words

Reply:
sure, i guess we should test all transformer encoders with sentences

Reply:
testing with sentences now